src/session.rs

```diff
@@ -281,7 +281,7 @@ impl Session {
             Cookies::Map(_) => unreachable!(),
             Cookies::List(cookies) => cookies.iter().map(|cookie| {
                 let mut cookie_builder =
-                    cookie_store::RawCookie::build(cookie.name.clone(), cookie.value.clone());
+                    cookie_store::RawCookie::build(cookie.name.clone());
```

Comment:
this was changed in the 0.21.1 release of cookie_store so required this modification, not entirely sure here if I'm doing it correctly. 😕

Reply:
The tests fail so looks like this is wrong yeah. According to the [docs](https://docs.rs/cookie_store/0.21.1/cookie_store/struct.RawCookie.html#method.build) we only need to turn the argument into a tuple:
```suggestion
                    cookie_store::RawCookie::build((cookie.name.clone(), cookie.value.clone()));
```

Reply:
@blyxxyz thank you for looking into this. I think while compiling I'm getting this deprecation warning via clippy which will fail the CI. 

```bash
warning: use of deprecated method `cookie::builder::CookieBuilder::<'c>::finish`: `CookieBuilder` can be passed in to methods expecting a `Cookie`; for other cases, use `CookieBuilder::build()`
   --> src/session.rs:308:37
    |
308 |                     &cookie_builder.finish(),
    |                                     ^^^^^^
    |
    = note: `#[warn(deprecated)]` on by default
```

Reply:
`&cookie_builder.into()` works.